pkgs/by-name/si/signal-export/package.nix

```diff
@@ -6,16 +6,17 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "signal-export";
-  version = "1.8.2";
+  version = "3.2.0";
   pyproject = true;
 
   src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-Hm0BVF2RUsxDacsAB3MJtk1t9FYmBPjeb5JzwaLkZ14=";
+    inherit version;
+    pname = "signal_export";
+    hash = "sha256-b0oJXUbJ9545oVMXBcM4MFjK3U3RiAT1lv9NUu6gF04=";
   };
 
   nativeBuildInputs = with python3.pkgs; [
-    setuptools-scm
+    pdm-backend
   ];
 
   propagatedBuildInputs = with python3.pkgs; [
```

Comment:
we should be able to drop setuptools here, now that the package uses pdm-backend

Reply:
:+1: I remove it from the BuildInputs as well.